src/Tasks/Al.cs

```diff
@@ -387,6 +392,131 @@ public override bool Execute()
             return base.Execute();
         }
 
+        #endregion
+    }
+
+#else
+
+    /// <summary>
+    /// Stub AL task for .NET Core
+    /// </summary>
+    public class AL : TaskRequiresFramework, IALTaskContract
```

Comment:
Please make the netcore version `sealed`--otherwise it can be derived from and someone can call `TaskExtension.Log` or other things and get _very_ confused.

Reply:
They should have been sealed. I had two classes sealed in the original commit. I only missed most of the classes. :roll_eyes:

Reply:
Change committed